### ISSUE TITLE
`pac solution` commands reach 100% too early

### ISSUE THREAD

**Issue by filcole** [microsoft/powerplatform-vscode#194]
When performing a solution import or export with `--async` the percentage complete will reach 100 before the platform actually completes the import/export.   e.g.

```
Processing asynchronous request... execution time: 00:06:20.3526315 and 95% complete
Processing asynchronous request... execution time: 00:06:24.3895085 and 96% complete
Processing asynchronous request... execution time: 00:06:28.4275202 and 97% complete
Processing asynchronous request... execution time: 00:06:32.4668427 and 98% complete
Processing asynchronous request... execution time: 00:06:36.5047753 and 99% complete
Processing asynchronous request... execution time: 00:06:40.5428450 and 100% complete
Processing asynchronous request... execution time: 00:06:44.5791273 and 100% complete
Processing asynchronous request... execution time: 00:06:48.6194272 and 100% complete
[ ... snip ... ]
Processing asynchronous request... execution time: 00:07:04.7814496 and 100% complete
Processing asynchronous request... execution time: 00:07:08.8193583 and 100% complete
[ ... snip ... ]
Processing asynchronous request... execution time: 00:10:46.9911967 and 100% complete
Processing asynchronous request... execution time: 00:10:51.0304897 and 100% complete
...
```

It's a cosmetic issue, and it's been around since `--async` was introduced.  A better approximation of 'completeness' would be reassuring though!

Also, thanks for the continued improvements and developments with pac.

**Comment by tehcrashxor** [comment]
Tracking as AB#2700452

**Comment by filcole** [comment]
The issue also occurs with the new --async flag on pac solution publish

**Comment by tehcrashxor** [comment]
The fix has been rolled out with PAC v1.16.6 / powerplatform-vscode v1.0.16 in #216
